addons/mail/static/tests/chat_window/chat_window.test.js

```diff
@@ -1095,7 +1095,7 @@ test("mark as read when opening chat window", async () => {
     await click(".o-mail-NotificationItem", { text: "bob" });
     await contains(".o-mail-ChatWindow .o-mail-ChatWindow-header", { text: "bob" });
     // composer is focused by default, we remove that focus
-    await contains(".o-mail-Composer-input", { setFocus: false });
+    document.querySelector(".o-mail-Composer-input").blur();
```

Comment:
Isn't there a chance that we blur the element before it gets focused then?
Shouldn't we have a contains with focus beforehand? Or even add blur into the helper?
Do we need to blur specifically, or could be click on body or something else, which is a more natural user action?

Reply:
Clicking on body sometimes fail, blur does not. Thus I'd rather keep it. I added assertions to ensure composer is focused beforehand and blurred afterwards.